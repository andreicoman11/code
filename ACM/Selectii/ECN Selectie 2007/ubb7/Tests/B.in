1
50.0
-100.000000 -100.000000 25.000000 0.000000
1
50.0
-100.000000 -100.000000 25.000000 0.392699
1
50.0
-100.000000 -100.000000 25.000000 0.785398
1
50.0
-100.000000 -100.000000 25.000000 1.178097
1
50.0
-100.000000 -100.000000 25.000000 1.570796
1
50.0
-100.000000 -100.000000 25.000000 1.963495
1
50.0
-100.000000 -100.000000 25.000000 2.356194
1
50.0
-100.000000 -100.000000 25.000000 2.748894
1
50.0
-100.000000 -100.000000 25.000000 3.141593
1
50.0
-100.000000 -100.000000 25.000000 3.534292
1
50.0
-100.000000 -100.000000 25.000000 3.926991
1
50.0
-100.000000 -100.000000 25.000000 4.319690
1
50.0
-100.000000 -100.000000 25.000000 4.712389
1
50.0
-100.000000 -100.000000 25.000000 5.105088
1
50.0
-100.000000 -100.000000 25.000000 5.497787
1
50.0
-100.000000 -100.000000 25.000000 5.890486
1
50.0
-100.000000 0.000000 25.000000 0.000000
1
50.0
-100.000000 0.000000 25.000000 0.392699
1
50.0
-100.000000 0.000000 25.000000 0.785398
1
50.0
-100.000000 0.000000 25.000000 1.178097
1
50.0
-100.000000 0.000000 25.000000 1.570796
1
50.0
-100.000000 0.000000 25.000000 1.963495
1
50.0
-100.000000 0.000000 25.000000 2.356194
1
50.0
-100.000000 0.000000 25.000000 2.748894
1
50.0
-100.000000 0.000000 25.000000 3.141593
1
50.0
-100.000000 0.000000 25.000000 3.534292
1
50.0
-100.000000 0.000000 25.000000 3.926991
1
50.0
-100.000000 0.000000 25.000000 4.319690
1
50.0
-100.000000 0.000000 25.000000 4.712389
1
50.0
-100.000000 0.000000 25.000000 5.105088
1
50.0
-100.000000 0.000000 25.000000 5.497787
1
50.0
-100.000000 0.000000 25.000000 5.890486
1
50.0
-100.000000 100.000000 25.000000 0.000000
1
50.0
-100.000000 100.000000 25.000000 0.392699
1
50.0
-100.000000 100.000000 25.000000 0.785398
1
50.0
-100.000000 100.000000 25.000000 1.178097
1
50.0
-100.000000 100.000000 25.000000 1.570796
1
50.0
-100.000000 100.000000 25.000000 1.963495
1
50.0
-100.000000 100.000000 25.000000 2.356194
1
50.0
-100.000000 100.000000 25.000000 2.748894
1
50.0
-100.000000 100.000000 25.000000 3.141593
1
50.0
-100.000000 100.000000 25.000000 3.534292
1
50.0
-100.000000 100.000000 25.000000 3.926991
1
50.0
-100.000000 100.000000 25.000000 4.319690
1
50.0
-100.000000 100.000000 25.000000 4.712389
1
50.0
-100.000000 100.000000 25.000000 5.105088
1
50.0
-100.000000 100.000000 25.000000 5.497787
1
50.0
-100.000000 100.000000 25.000000 5.890486
1
50.0
0.000000 -100.000000 25.000000 0.000000
1
50.0
0.000000 -100.000000 25.000000 0.392699
1
50.0
0.000000 -100.000000 25.000000 0.785398
1
50.0
0.000000 -100.000000 25.000000 1.178097
1
50.0
0.000000 -100.000000 25.000000 1.570796
1
50.0
0.000000 -100.000000 25.000000 1.963495
1
50.0
0.000000 -100.000000 25.000000 2.356194
1
50.0
0.000000 -100.000000 25.000000 2.748894
1
50.0
0.000000 -100.000000 25.000000 3.141593
1
50.0
0.000000 -100.000000 25.000000 3.534292
1
50.0
0.000000 -100.000000 25.000000 3.926991
1
50.0
0.000000 -100.000000 25.000000 4.319690
1
50.0
0.000000 -100.000000 25.000000 4.712389
1
50.0
0.000000 -100.000000 25.000000 5.105088
1
50.0
0.000000 -100.000000 25.000000 5.497787
1
50.0
0.000000 -100.000000 25.000000 5.890486
1
50.0
0.000000 0.000000 25.000000 0.000000
1
50.0
0.000000 0.000000 25.000000 0.392699
1
50.0
0.000000 0.000000 25.000000 0.785398
1
50.0
0.000000 0.000000 25.000000 1.178097
1
50.0
0.000000 0.000000 25.000000 1.570796
1
50.0
0.000000 0.000000 25.000000 1.963495
1
50.0
0.000000 0.000000 25.000000 2.356194
1
50.0
0.000000 0.000000 25.000000 2.748894
1
50.0
0.000000 0.000000 25.000000 3.141593
1
50.0
0.000000 0.000000 25.000000 3.534292
1
50.0
0.000000 0.000000 25.000000 3.926991
1
50.0
0.000000 0.000000 25.000000 4.319690
1
50.0
0.000000 0.000000 25.000000 4.712389
1
50.0
0.000000 0.000000 25.000000 5.105088
1
50.0
0.000000 0.000000 25.000000 5.497787
1
50.0
0.000000 0.000000 25.000000 5.890486
1
50.0
0.000000 100.000000 25.000000 0.000000
1
50.0
0.000000 100.000000 25.000000 0.392699
1
50.0
0.000000 100.000000 25.000000 0.785398
1
50.0
0.000000 100.000000 25.000000 1.178097
1
50.0
0.000000 100.000000 25.000000 1.570796
1
50.0
0.000000 100.000000 25.000000 1.963495
1
50.0
0.000000 100.000000 25.000000 2.356194
1
50.0
0.000000 100.000000 25.000000 2.748894
1
50.0
0.000000 100.000000 25.000000 3.141593
1
50.0
0.000000 100.000000 25.000000 3.534292
1
50.0
0.000000 100.000000 25.000000 3.926991
1
50.0
0.000000 100.000000 25.000000 4.319690
1
50.0
0.000000 100.000000 25.000000 4.712389
1
50.0
0.000000 100.000000 25.000000 5.105088
1
50.0
0.000000 100.000000 25.000000 5.497787
1
50.0
0.000000 100.000000 25.000000 5.890486
1
50.0
100.000000 -100.000000 25.000000 0.000000
1
50.0
100.000000 -100.000000 25.000000 0.392699
1
50.0
100.000000 -100.000000 25.000000 0.785398
1
50.0
100.000000 -100.000000 25.000000 1.178097
1
50.0
100.000000 -100.000000 25.000000 1.570796
1
50.0
100.000000 -100.000000 25.000000 1.963495
1
50.0
100.000000 -100.000000 25.000000 2.356194
1
50.0
100.000000 -100.000000 25.000000 2.748894
1
50.0
100.000000 -100.000000 25.000000 3.141593
1
50.0
100.000000 -100.000000 25.000000 3.534292
1
50.0
100.000000 -100.000000 25.000000 3.926991
1
50.0
100.000000 -100.000000 25.000000 4.319690
1
50.0
100.000000 -100.000000 25.000000 4.712389
1
50.0
100.000000 -100.000000 25.000000 5.105088
1
50.0
100.000000 -100.000000 25.000000 5.497787
1
50.0
100.000000 -100.000000 25.000000 5.890486
1
50.0
100.000000 0.000000 25.000000 0.000000
1
50.0
100.000000 0.000000 25.000000 0.392699
1
50.0
100.000000 0.000000 25.000000 0.785398
1
50.0
100.000000 0.000000 25.000000 1.178097
1
50.0
100.000000 0.000000 25.000000 1.570796
1
50.0
100.000000 0.000000 25.000000 1.963495
1
50.0
100.000000 0.000000 25.000000 2.356194
1
50.0
100.000000 0.000000 25.000000 2.748894
1
50.0
100.000000 0.000000 25.000000 3.141593
1
50.0
100.000000 0.000000 25.000000 3.534292
1
50.0
100.000000 0.000000 25.000000 3.926991
1
50.0
100.000000 0.000000 25.000000 4.319690
1
50.0
100.000000 0.000000 25.000000 4.712389
1
50.0
100.000000 0.000000 25.000000 5.105088
1
50.0
100.000000 0.000000 25.000000 5.497787
1
50.0
100.000000 0.000000 25.000000 5.890486
1
50.0
100.000000 100.000000 25.000000 0.000000
1
50.0
100.000000 100.000000 25.000000 0.392699
1
50.0
100.000000 100.000000 25.000000 0.785398
1
50.0
100.000000 100.000000 25.000000 1.178097
1
50.0
100.000000 100.000000 25.000000 1.570796
1
50.0
100.000000 100.000000 25.000000 1.963495
1
50.0
100.000000 100.000000 25.000000 2.356194
1
50.0
100.000000 100.000000 25.000000 2.748894
1
50.0
100.000000 100.000000 25.000000 3.141593
1
50.0
100.000000 100.000000 25.000000 3.534292
1
50.0
100.000000 100.000000 25.000000 3.926991
1
50.0
100.000000 100.000000 25.000000 4.319690
1
50.0
100.000000 100.000000 25.000000 4.712389
1
50.0
100.000000 100.000000 25.000000 5.105088
1
50.0
100.000000 100.000000 25.000000 5.497787
1
50.0
100.000000 100.000000 25.000000 5.890486
1
74.562902
163.831206 78.465587 15.796903 3.728309
1
56.616694
-19.820880 -92.154476 6.034416 1.311602
1
96.648471
84.600067 51.829295 22.230120 5.693742
2
94.902959
132.039101 46.744919 25.755535 4.804393
-79.356185 166.179018 36.161305 5.036268
2
78.343465
178.314822 96.305342 24.493895 2.864982
-117.779907 -115.629014 10.730990 3.456707
2
59.676865
196.020508 168.866633 24.387815 5.604564
113.531564 -72.016567 4.373471 4.358629
3
51.105780
40.972128 159.354743 2.390980 5.917550
-102.891578 -82.975822 11.787129 5.241488
99.016215 -161.915329 20.219837 4.751798
3
86.481493
32.348369 175.825071 18.249142 5.301764
-93.252553 -132.130025 27.221519 5.635917
-80.955536 178.573830 41.672735 2.007243
3
78.392102
-198.103189 -178.549795 4.953200 1.488021
83.456853 120.664055 7.997931 1.179597
57.352868 -52.884761 34.696901 1.197631
4
88.168784
-156.910266 49.375834 24.719668 0.767341
-53.115179 125.031305 26.072171 4.115289
25.662489 -32.171220 16.791502 1.620718
-29.416696 -30.659601 35.986102 1.870705
4
73.738270
-191.627823 -194.817869 31.079253 1.020094
21.816790 -112.500018 4.777893 3.112658
-114.470004 26.569036 31.239052 6.081060
-83.949132 6.228038 27.754797 1.379379
4
74.874563
175.346286 -111.193274 1.371813 6.156762
-170.318736 -56.218461 22.494947 5.416417
67.134748 159.779882 1.563950 1.974125
65.170389 138.272450 26.590421 4.021693
5
56.408412
57.113324 -183.026651 21.639798 5.823139
-6.448872 56.260523 12.976859 0.603773
-181.155165 -121.036617 18.851902 4.462015
44.123580 -159.566611 1.174143 5.539021
-188.127646 -19.326146 21.732348 4.198602
5
74.444460
-70.806175 -163.146260 6.501701 2.843490
58.662291 27.445086 35.295492 5.970592
50.204859 49.197944 12.195476 4.411322
-61.254766 -131.286813 16.542366 5.712650
190.096291 58.216368 24.525661 4.878518
5
58.263506
-119.653077 51.696056 19.692974 5.239156
176.851733 -128.084566 20.518903 0.768107
171.687484 -44.554947 19.628289 2.421394
-109.806999 -149.335672 22.160938 1.215649
-10.318014 -69.537658 15.194672 1.503082
6
89.238452
-6.651715 -111.651015 32.910721 0.136815
-186.073377 118.922328 19.326142 5.770015
-176.906537 -128.440900 14.595537 1.278779
80.063158 -182.613500 25.608741 4.329688
124.236926 118.698770 24.529941 5.991518
-14.972420 -159.013246 1.000000 0.227099
6
86.687265
142.495665 -98.769451 18.598954 3.634267
-159.516194 -158.352935 8.295072 6.210784
35.650906 -83.429848 35.416253 5.759510
-95.681800 61.034678 1.897430 6.148776
-127.509394 94.178283 10.281467 2.609969
119.837902 -24.627647 18.423927 3.702548
6
62.888595
-114.413419 -33.063666 10.159059 1.192771
-139.653859 189.640837 11.304265 1.464247
12.231301 -112.080573 22.415745 1.430219
-41.879986 -2.976427 13.908406 6.260924
92.719400 176.253500 18.216968 0.612406
-79.519422 -0.377678 8.039301 2.414164
7
89.378637
-43.231391 39.849463 29.693787 4.381767
-132.186572 -158.079645 42.745336 1.929661
149.745188 114.661225 4.789523 1.704259
-147.444333 133.144441 28.087251 3.862428
82.922573 193.564337 32.904692 5.498259
50.761983 23.992064 38.317985 3.801904
11.270623 -55.012052 9.736769 5.466927
7
85.605501
-52.608407 -99.041840 22.888816 3.209660
157.282701 -43.310903 34.029888 0.221344
-91.551764 61.387684 14.000558 0.114086
-79.688773 8.872639 38.742109 4.786839
180.222065 -156.677733 38.483833 6.021983
-147.217100 169.319029 30.093079 4.487708
-73.333644 129.292569 2.652947 3.275703
8
64.921490
-88.082880 -161.410032 10.522355 3.754327
138.389267 25.657848 10.344294 3.163445
-190.040782 -170.318805 18.245659 3.304199
16.505223 82.407213 13.059284 4.192431
-165.176946 129.464402 20.237778 1.358309
89.286262 15.068724 14.990342 0.023159
-76.075907 -108.369945 17.367081 5.904397
95.838210 -129.673423 8.524169 0.036196
8
73.593808
28.680356 153.284016 15.380627 1.570680
116.884211 -117.241550 1.408396 2.315270
76.409163 -167.035974 21.133386 3.905637
-69.757183 -29.824897 34.541281 5.600979
-139.813392 -97.262726 7.434090 0.315345
59.231532 92.531082 8.651697 4.217214
-155.279937 121.886269 24.521773 3.265251
-96.387862 -24.063624 10.342300 1.144689
0
