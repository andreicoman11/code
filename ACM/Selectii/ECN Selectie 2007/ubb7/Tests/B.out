7
6
6
6
7
8
10
12
15
16
17
16
15
12
10
8
4
4
5
6
7
9
10
12
12
12
10
9
7
6
5
4
7
8
10
12
15
16
17
16
15
12
10
8
7
6
6
6
7
6
5
4
4
4
5
6
7
9
10
12
12
12
10
9
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
7
9
10
12
12
12
10
9
7
6
5
4
4
4
5
6
15
12
10
8
7
6
6
6
7
8
10
12
15
16
17
16
12
12
10
9
7
6
5
4
4
4
5
6
7
9
10
12
15
16
17
16
15
12
10
8
7
6
6
6
7
8
10
12
12
16
6
7
20
34
72
28
55
8
25
157
147
30
21
161
60
34
43
58
68
118
